orm/models.py

```diff
@@ -95,14 +95,22 @@ def build_select_expression(self):
         return expr
 
     def filter(self, **kwargs):
+        return self._parse_filters(kwargs)
+
+    def exclude(self, **kwargs):
+        return self._parse_filters(kwargs, exclude=True)
+
+    def _parse_filters(self, filters, exclude=False):
```

Comment:
Doesn't seem like this name describes what happens within the function. How about something like this:

```suggestion
    def _filter_query(self, filters, exclude=False):
```